nautobot/docs/rest-api/overview.md

```diff
@@ -785,3 +785,478 @@ http://nautobot/api/dcim/sites/ \
+        "parent": null,
+        "location_type": "e3d4a9af-c6c1-4582-b483-a13301eb6e28",
+        "tenant": "5b1feadb-fab0-4f81-a53f-5192d83b0216",
+        "tags": [
+            "6e8ce6c9-0a8c-4731-b02a-b2ec19db5c52",
+            "1e986e5d-099d-4742-bfa5-16363abea5fd",
+            "21afbdad-c782-4fc5-9b3b-7b59273bb08c",
+            "0118b994-9943-4295-946b-92b27efe15db"
+        ]
+    },
+    "location_type": {
+        "id": "4edcc111-e3f7-4309-ab0e-eb34c001874e",
+        "display": "Campus → Building",
+        "url": "http://nautobot/api/dcim/location-types/4edcc111-e3f7-4309-ab0e-eb34c001874e/",
+        "custom_fields": {},
+        "notes_url": "http://nautobot/api/dcim/location-types/4edcc111-e3f7-4309-ab0e-eb34c001874e/notes/",
+        "tree_depth": null,
+        "content_types": [
+            "dcim.rack",
+            "ipam.prefix",
+            "ipam.vlan",
+            "dcim.rackgroup",
+            "ipam.vlangroup"
+        ],
+        "created": "2023-04-12T19:29:06.707759Z",
+        "last_updated": "2023-04-12T19:29:06.716482Z",
+        "name": "Building",
+        "slug": "building",
+        "description": "Protect growth bill all hair along.",
+        "nestable": false,
+        "parent": "e3d4a9af-c6c1-4582-b483-a13301eb6e28"
+    },
+    "tenant": null,
+    "tags": []
+}
+```
+
+### ?depth=2 and beyond
+
+For `TreeModels`, sometimes it is necessary to traverse up more than one tree level and to retrieve an ancestor's information that is not a direct parent of the object.
+In this case, `?depth=2` and beyond would become very useful.
+
+!!! note
+    Notice that the location and its location_type's parent's parent's serializers are shown as well.
+
+```no-highlight
+curl -s -X GET \
+-H "Accept: application/json; version=2.0" \
+http://nautobot/api/dcim/locations/3b71a669-faa4-4f8d-a72a-8c94d121b793/?depth=2 | jq '.'
+```
+
+```json
+{
+    "id": "3b71a669-faa4-4f8d-a72a-8c94d121b793",
+    "display": "Campus-01 → Building-02 → Elevator-46",
+    "url": "http://nautobot/api/dcim/locations/3b71a669-faa4-4f8d-a72a-8c94d121b793/",
+    "custom_fields": {
+        "example_plugin_auto_custom_field": null
+    },
+    "notes_url": "http://nautobot/api/dcim/locations/3b71a669-faa4-4f8d-a72a-8c94d121b793/notes/",
+    "tree_depth": 2,
+    "time_zone": null,
+    "circuit_count": 0,
+    "device_count": 0,
+    "prefix_count": 0,
+    "rack_count": 0,
+    "virtual_machine_count": 0,
+    "vlan_count": 0,
+    "created": "2023-04-12T19:29:09.635967Z",
+    "last_updated": "2023-04-12T19:29:09.648995Z",
+    "name": "Elevator-46",
+    "slug": "campus-01-building-02-elevator-46",
+    "description": "Thank century single same.",
+    "facility": "",
+    "asn": null,
+    "physical_address": "",
+    "shipping_address": "",
+    "latitude": "-49.390000",
+    "longitude": null,
+    "contact_name": "",
+    "contact_phone": "+1-645-720-2291x878",
+    "contact_email": "",
+    "comments": "",
```

Comment:
Might make the example more readable/clear if we summarize `...` the parts of the response that are unchanged from the previous example?